itl/command-plugins.conf

```diff
@@ -549,6 +549,182 @@ object CheckCommand "http" {
 	vars.http_verbose = false
 }
 
+object CheckCommand "curl" {
+	import "ipv4-or-ipv6"
+
+	command = [ PluginDir + "/check_curl" ]
+
+	arguments += {
+		"-H" = {
+			description = "Host name argument for servers using host headers (virtual host). Append a port to include it in the header (eg: example.com:5000)"
+			value = "$curl_vhost$"
+		}
+		"-I" = {
+			description = "IP address or name (use numeric address if possible to bypass DNS lookup)."
+			value = "$curl_ip$"
+		}
+		"-p" = {
+			description = "Port number (default: 80)"
+			value = "$curl_port$"
+		}
+		"-S" = {
+			description = "Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation (2 = SSLv2, 3 = SSLv3, 1 = TLSv1, 1.1 = TLSv1.1, 1.2 = TLSv1.2, 1.3 = TLSv1.3). With a '+' suffix, newer versions are also accepted. Note: SSLv2 and SSLv3 are deprecated and are usually disabled in libcurl"
+			set_if = "$curl_tls$"
+			value = "$curl_tls_version$"
+		}
+		"--sni" = {
+			description = {{{Enable SSL/TLS hostname extension support (SNI)
+Note: --sni is the default in libcurl as SSLv2 and SSLV3 are deprecated and
+SNI only really works since TLSv1.0}}}
```

Comment:
You don't need to do that, just put it in regular quotation marks, please! and without the specification `Note:` , just like a normal sentence. Please adjust all other similar descriptions as well!

Reply:
done